src/mountutils.cpp

```diff
@@ -22,4 +24,12 @@ NAN_MODULE_INIT(MountUtilsInit) {
   .ToLocalChecked());
 }
 
+
+void MountUtilsLog(std::string string) {
+  const char* debug = std::getenv("MOUNTUTILS_DEBUG");
```

Comment:
If we used `DEBUG=true` or `DEBUG=mountutils` rather than `MOUNTUTILS_DEBUG` I guess that makes it harder to debug Etcher without also debugging mountutils, or vice-versa?